userguide/contribute.rst

```diff
@@ -0,0 +1,179 @@
+.. _Contributing to %brand%:
+
+Contributing to %brand%
+=================================
+
+%brand% is an open source community, relying on the input and
+expertise of its users to help grow and improve %brand%. When users
```

Comment:
Please avoid possessives, especially "its".  Here, it can just be removed.  So can the redundant use of %brand% at the end of the sentence.  For that matter, this is one of the few places we don't use %brand%, because this section is specific to FreeNAS.  So:
"FreeNAS(R markup) is an open source community, relying on the input and
expertise of users to grow and improve."